pages/doc/trace_data_details.md

```diff
@@ -88,13 +88,44 @@ getAllUsers source=localhost traceId=7b3bf470-9456-11e8-9eb6-529269fb1459 spanId
 
 ### Span Tags
 
-Span tags are special tags associated with a span.
+Span tags are special tags associated with a span and are key value pairs.
```

Comment:
Do we also say point tags are key value pairs? If not, should we?

Reply:
We mention it "Point tags are key-value pairs (strings) that are associated with a point." I will use the same format.